openedx/features/edly_panel/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+# Create your models here.
+class EdlyActiveUser(models.Model):
```

Comment:
We should rename this to `EdlyUserActivity`.

Reply:
done.